lms/djangoapps/mobile_api/testutils.py

```diff
@@ -10,15 +10,16 @@
      MobileCourseAccessTestMixin - tests for APIs with mobile_course_access.
 """
 # pylint: disable=no-member
-from datetime import timedelta
 
 import ddt
 import datetime
+from datetime import timedelta
```

Comment:
Is this used? Could you use `datetime.timedelta` and not double-import?

Reply:
@sanfordstudent Thy feedback is gratefully accepted.